Packs/MicrosoftTeams/Integrations/MicrosoftTeams/README.md

```diff
@@ -1123,15 +1110,15 @@ Note: To enrich an incident created via the Demisto BOT (`new incident` command)
         - From the Cortex XSOAR machine to localhost.
           - Note: The web server supports only POST method queries. 
         
-   2. If the cURL queries were sent successfully, you should see the following line in Cortex XSOAR logs: `Finished processing Microsoft Teams activity successfully`.
+   3. If the cURL queries were sent successfully, you should see the following line in Cortex XSOAR logs: `Finished processing Microsoft Teams activity successfully`.
 
-   3. If you're working with secured communication (HTTPS), make sure that you provided a valid certificate.
+   4. If you're working with secured communication (HTTPS), make sure that you provided a valid certificate. (Not XSOAR/XSIAM Rerouting ).
        1. Run `openssl s_client -connect <domain.com>:443` .
        2. Verify that the returned value of the `Verify return code` field is `0 (ok)`, otherwise, it's not a valid certificate.
     
-   4. Try inserting your configured message endpoint in a browser and click **Enter**. If `Method Not Allowed` is returned, the endpoint is valid and ready to communicate, otherwise, it needs to be handled according to the returned error's message.
+   5. Try inserting your configured message endpoint in a browser and click **Enter**. If `Method Not Allowed` is returned, the endpoint is valid and ready to communicate, otherwise, it needs to be handled according to the returned error's message. (Not for XSOAR 8 OR XSIAM).
```

Comment:
```suggestion
   5. Try inserting your configured message endpoint in a browser and click **Enter**. If `Method Not Allowed` is returned, the endpoint is valid and ready to communicate, otherwise, it needs to be handled according to the returned error's message. (Not for Cortex XSOAR 8 OR Cortex XSIAM).
```